src/external/win_guess_utils.jl

```diff
@@ -0,0 +1,145 @@
+@doc raw"""
+    The quantum numbers given by Wannier90 are not in common use. 
+    In turn the order in which the orbitals s,p,d and f are given in the Tables 3
+    (see [Wannier90's user guide][http://www.wannier.org/support/] p.54) 
+    is not matching the classic order given in spherical_harmonics.jl. 
+   
+    The purpose of this function is to retrieve the proper quantum number m
+    from the one given in the nnkp file.
+
+    Corresponding m Wannier <-> DFTK:  p  [1,2,3]         <->  [0,1,-1]
+                                       d  [1,2,3,4,5]     <->  [0,1,-1,2,-2]
+                                       f  [1,2,3,4,5,6,7] <->  [0,1,-1,2,-2,3,-3]
+"""
+function retrieve_proper_m(l::Integer,mr::Integer)
+    @assert 0 ≤ l
+    m = zero(Int64)
+    m_p = [0,1,-1]; m_d = [0,1,-1,2,-2]; m_f = [0,1,-1,2,-2,3,-3]
+    (l == 0) && return 0
+    (l == 1) && return m_p[mr]  # p
+    (l == 2) && return m_d[mr]  # d
+    (l == 3) && return m_f[mr]  # f
+    error("Quantum numbers are not matching any implemented
+                orbital (s,p,d,f)")
+end
+
+@doc raw"""
+    Gives the analytic expression of the integral 
+    ``I_l(G) = \int_{\mathbb{R}^+} r^(l+2) exp(-r^2/2) j_l(|G|r)dr``
+    as given in arXiv:1908.07374v2 
```

Comment:
pretty sure that's standard and there's no need to go into such a specific reference, but OK to keep it as it is. Add reference to the specific equation (since that paper is about a much more complicated problem)